src/main/java/com/onelogin/sdk/conn/Client.java

```diff
@@ -2739,7 +2741,7 @@ public List<OTPDevice> getEnrolledFactors(long userId) throws OAuthSystemExcepti
      * @see <a target="_blank" href="https://developers.onelogin.com/api-docs/1/multi-factor-authentication/activate-factor">Activate an Authentication Factor documentation</a>
      * @see <a target="_blank" href="https://developers.onelogin.com/api-docs/2/multi-factor-authentication/activate-factor">Activate an Authentication Factor documentation</a>
      */
-    public FactorEnrollmentResponse activateFactor(long userId, String deviceId, Integer expiresIn, String customMessage) throws OAuthSystemException, OAuthProblemException, URISyntaxException, ErrorResourceInitialization
+    public FactorEnrollmentResponse activateFactor(long userId, String deviceId, Integer expiresIn, String redirectTo, String customMessage) throws OAuthSystemException, OAuthProblemException, URISyntaxException, ErrorResourceInitialization
```

Comment:
I would move this to the end since this will break things for anyone currently using customMessage.

Reply:
I agree